### ISSUE TITLE
Empty state button on istio.io homepage

### ISSUE THREAD

**Issue by Lealikeintea** [istio/istio.io#7428]
There is an invisible button on the lower eighth of the istio.io homepage with no text content or alt-text, violating two WCAG accessibility guidelines (linked below). While the button is not visible, it is present to screen readers and keyboard navigation, and can create confusion when navigating through the site.

[1.1.1 Non-text Content (Level A)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=243#non-text-content)
[2.4.4 Link Purpose (In Context) (Level A)](w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=243#link-purpose-in-context)

**Comment by pizzaz93** [comment]
Additionally when tab selecting `Increasing Kubernetes deployment and management efficiency` and the Learn More button, there isn't any indication that the button is in focus.
![Screenshot 2022-05-13 10 45 00 AM](https://user-images.githubusercontent.com/10904763/168339134-12c02572-5ecc-4766-a5a2-aae9ca81ea6c.png)
)
